homeassistant/components/epson/config_flow.py

```diff
@@ -24,18 +25,22 @@
 class EpsonConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for epson."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
```suggestion
    VERSION = 1
    MINOR_VERSION = 2
```

We should increase the minor version instead, as we were adding a new field.
Please adopt the rest of the code